cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -37,155 +23,123 @@
     "sex": {
     },
     "tissue": {
+        "CL:0010003": "CL:0000322", # AUTOMATED
     },
 }
 
-DEPRECATED_FEATURE_IDS = [
-]
-
-# Dictionary for CURATOR-DEFINED remapping of deprecated feature IDs, if any, to new feature IDs.
-GENCODE_MAPPER = {}
+df = pd.read_csv('migrate_files/non_csr_list.csv')
```

Comment:
No need for a csv as it's a single columns. So remove the column header, and you can just read into a list without going through pandas